### PR TITLE
Add Arm job adds

### PR DESCRIPTION
**Pull request by Kislovskiy**
Adding jobs adds from arm. Source:
[Arm Job Board Ads for Euro Python 2023.docx](https://github.com/EuroPython/website/files/12003439/Arm.Job.Board.Ads.for.Euro.Python.2023.docx)

### CODE REVIEW

File: data/pages-content/jobs-board.md [hunk 1]
```diff
@@ -175,11 +175,62 @@ Temporal helps developers focus on the important things, like writing business l
...
+<details>
+  <summary><h4 id="arm1-compiler-engineer"> Compiler Engineer</h4></summary>
+
+  Arm Machine Learning (ML) Compilers team is looking for a software engineer who would build a range of innovative compiler solutions for a variety of markets.
+
+  You will apply your experience and insight within this domain to craft and optimise compilers for machine learning networks that target range of Arm’s CPUs, GPUs and NPUs.
+
+  If you are interested in this opportunity, make sure to apply soon! We look forward to receiving your application and welcoming you to Arm. You could be joining our highly motivated compiler team and have a marked impact on both strategy and implementation!
+
+  Responsibilities:
```
Comment (hypha) [inline data/pages-content/jobs-board.md:199]:
```suggestion
  **Responsibilities:**
```

File: data/pages-content/jobs-board.md [hunk 2]
```diff
@@ -175,11 +175,62 @@ Temporal helps developers focus on the important things, like writing business l
...
+</details>
+
+[Apply here](https://careers.arm.com/job/cambridge/compiler-engineer/33099/49643515840?utm_source=Euro%20Python%202023%20-%20Compiler%20Engineer%202023-8356&utm_medium=Euro%20Python%202023%20-%20Compiler%20Engineer%202023-8356&utm_campaign=Euro%20Python%202023%20-%20Compiler%20Engineer%202023-8356)
+
+<details>
+  <summary><h4 id="arm2-software-engineer"> Software Engineer</h4></summary>
+
+Do you want to help build tomorrow's solutions? This is an excellent opportunity to work in the Architecture and Future Technologies team in Arm's Development Solutions Group in Cambridge, UK. We investigate and prototype new tooling technologies to support Arm's evolving architecture and product roadmap. We find out how to help new features land successfully in the software ecosystem, and we feed back to architects and hardware design teams.
+
+Responsibilities:
```
Comment (hypha) [inline data/pages-content/jobs-board.md:211]:
```suggestion
**Responsibilities:**
```

**Review by hypha — COMMENTED** [review]
Some minimal changes based on the docx.

File: data/pages-content/jobs-board.md [hunk 3]
```diff
@@ -175,11 +175,62 @@ Temporal helps developers focus on the important things, like writing business l
 <details>
   <summary><h4 id="temporal1-devrel-engineer"> Developer Relations Engineer</h4></summary>
 
-  The Developer Relations Engineer will be Temporal’s strongest connection into the community, through their code demos utilizing Temporal and adjacent technologies, sharing that story through written content and verbalizing through talks. You will be an integral part of the success of Temporal’s customer journey and life cycle. Your biggest allies will be in Developer Education and Engineering, while partnering with Marketing on how the story evolves.  
+  The Developer Relations Engineer will be Temporal’s strongest connection into the community, through their code demos utilizing Temporal and adjacent technologies, sharing that story through written content and verbalizing through talks. You will be an integral part of the success of Temporal’s customer journey and life cycle. Your biggest allies will be in Developer Education and Engineering, while partnering with Marketing on how the story evolves.
 </details>
 
 [Apply here](https://jobs.lever.co/temporal/ff1b33fe-fd12-4d8c-a266-4b1a83295182)
 
+## [arm]
```
[the remark this answers is not in the document]
Comment (Kislovskiy) [reply data/pages-content/jobs-board.md]: let's point to their website

**Pull request merged**